src/main/java/com/cascadebot/cascadebot/commands/moderation/KickCommand.java

```diff
@@ -19,6 +19,14 @@ public void onCommand(Member sender, CommandContext context) {
             return;
         }
         Member targetMember = DiscordUtils.getMember(context.getMessage(0), context.getGuild());
+        if (targetMember.getUser() == sender.getUser()) {
+            context.replyWarning("Why would I kick you~");
+            return;
+        }
+        if (targetMember.getUser() == context.getSelfMember().getUser()) {
+            context.replyWarning("I'm not going to kick myself~");
```

Comment:
~~I'm sorry Dave, I'm afraid I can't let you do that~~ 

Maybe? Just a fun idea
```suggestion
            context.replyWarning("My programming forbids me to kick myself~");
```